packages/toolpad-app/src/PinholeOverlay.tsx

```diff
@@ -86,7 +86,7 @@ const PinholeOverlay = React.forwardRef(function PinholeOverlay(
 
   // We key the elements so that React doesn't reuse between pinhole off or on
   return (
-    <PinholeOverlayRoot ref={ref} className={className}>
+    <PinholeOverlayRoot ref={ref} className={className} data-testid="hud-overlay">
```

Comment:
This isn't really the overlay the the drag & drop interacts with - the overlay is `OverlayRoot` and it already has the test id `page-overlay` in case that also works.

Reply:
Thanks! that does work 👍